modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/resources/ExtensionModelJsonGeneratorTestCase.java

```diff
@@ -188,7 +163,7 @@ public void load() throws Exception {
   public static ExtensionModel loadExtension(Class<?> clazz, ExtensionModelLoader loader) {
     Map<String, Object> params = new HashMap<>();
     params.put(TYPE_PROPERTY_NAME, clazz.getName());
-    params.put(VERSION, MULE_VERSION);
+    params.put(VERSION, "4.0.0-SNAPSHOT");
```

Comment:
not ok. This is going to break the moment we branch for GA. Fix it in another PR though